ex6/ex6.c

```diff
@@ -14,13 +14,29 @@ and `clock_gettime()` should work just fine.
 #include <stdio.h>
 #include <unistd.h>
 #include <time.h>
+#include <stdint.h>
 
 #define number_iter 1000000
 #define BILLION 1000000000L
 
 int main()
 {
     // Your code here
-    
+    uint64_t diff;
+    struct timespec start, end;
+    int avg;
+    long total;
+
+    for (int i = 0; i < number_iter; i++)
+    {
+        clock_gettime(CLOCK_MONOTONIC, &start);
+        write(STDOUT_FILENO, NULL, 0);

```

Comment:
Great Job! you used STDOUT_FILENO which is an integer file descriptor. You could also use stdout but this solution is much simpler!